site/docs/data.md

```diff
@@ -71,6 +71,34 @@ For example, to create a data source named `myData`, use the following data
 }
 ```
 
+Then you can load it with vegaEmbed modying the [Vega View](https://vega.github.io/vega/docs/api/view/#data) using the insert or change methods.
+
+```js 
+const embed_opt = {"mode": "vega-lite"};
+const view = vegaEmbed("#vis", spec, embed_opt)
+      ...
+			.then((res) => res.view
+        .insert("myData", [ /* some data array */])
+        .run()
+      );
+```
+
+Or if you prefer avoiding vegaEmbed
```

Comment:
Let's remove this second example to avoid confusion.

Reply:
Instead, let's link to https://vega.github.io/vega-lite/tutorials/streaming.html as an example.

Reply:
Good idea.